db/migrate/20191217193112_update_water_control_metadata_field.rb

```diff
@@ -0,0 +1,19 @@
+# See SeedMetadataFields. This migration changes water control to be required.
+# NOTE: we should probably be using seeds.rb instead of migrations for initializing data.
+class UpdateWaterControlMetadataField < ActiveRecord::Migration[5.1]
+  def up
+    water_control = MetadataField.find_by(name: "water_control")
+    if water_control
+      # default_for_new_host_genome should already equal 1.
+      # See AddNewHostGenomeDefaultToMetadataFields.
+      water_control.update(is_required: 1)
+    end
+  end
+
+  def down
+    water_control = MetadataField.find_by(name: "water_control")
+    if water_control
+      water_control.update(is_required: 0)
+    end
+  end
+end
```

Comment:
It's not clear to me that this is the right approach to making water control "required", as opposed to simply running `MetadataField.find_by(name: "water_control").update(is_required: 1)` in the Rails console. Although I know this is how we've often done it in the past. 

As long as the application doesn't depend on water_control being required, this feels like something an admin would configure either in an admin dashboard or via the rails console, as opposed to something to put in a migration. What are your thoughts?

Reply:
I was just trying to follow the precedent of declaring the required metadata in migrations. 

I think the best approach is to use `seeds.rb` which is what that is designed for. Then we would have an easy to read document of built-in metadata. I'm just not sure how painful converting would be.